Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_breakfast.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity
+  id: RandomFoodMeal
+  name: random food spawner
+  suffix: Meal
+  parent: MarkerBase
+  placement:
+    mode: AlignTileAny
+  components:
+  - type: Sprite
+    layers:
+      - state: red
+      - sprite: Objects/Consumable/Food/meals.rsi
```

Comment:
Consider changing the sprite to one that corresponds to the spawner's contents (i.e. breakfast.rsi & one of the breakfasts)